### ISSUE TITLE
Incorrect role is define for 'View details' control.

### ISSUE THREAD

**Issue by JasonAment** [microsoft/AdaptiveCards#2240]
Please review the following bug for repro steps and video:
https://github.com/Microsoft/BotFramework-WebChat/issues/1350

**Comment by shalinijoshi19** [comment]
@dclaux wasn't able to repro this real quick. @corinagum is this somethign that is still repro'ing for your team?

**Comment by corinagum** [comment]
@shalinijoshi19, thanks for the ping. Asking @compulim to look into this one, he will get back to you. :)

**Comment by compulim** [comment]
![image](https://user-images.githubusercontent.com/1622400/79374316-1a8ced80-7f0c-11ea-962a-3fd4c31cf76c.png)

It is still repro.

AFAIK, if the action is a link, e.g. `Action.OpenUrl`, it should have `role="link"` instead of `role="button"`.

BTW, quick way to test AC in Web Chat (we use AC 1.2.5):

1. Navigate to https://compulim.github.io/webchat-loader/
1. Select "4.8.0"
1. Click on preset "[Public] MockBot"
1. Click "Open Web Chat in a new window"
1. Upload a JSON file with the AC

The AC card JSON we tested:

```json
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.0",
    "speak": "<s>Your  meeting about \"Adaptive Card design session\"<break strength='weak'/> is starting at 12:30pm</s><s>Do you want to snooze <break strength='weak'/> or do you want to send a late notification to the attendees?</s>",
    "body": [
        {
            "type": "TextBlock",
            "text": "Adaptive Card design session",
            "size": "Large",
            "weight": "Bolder"
        },
        {
            "type": "TextBlock",
            "text": "Conf Room 112/3377 (10)",
            "isSubtle": true
        },
        {
            "type": "TextBlock",
            "text": "12:30 PM - 1:30 PM",
            "isSubtle": true,
            "spacing": "None"
        },
        {
            "type": "TextBlock",
            "text": "Snooze for"
        },
        {
            "type": "Input.ChoiceSet",
            "id": "snooze",
            "value": "5",
            "choices": [
                {
                    "title": "5 minutes",
                    "value": "5"
                },
                {
                    "title": "15 minutes",
                    "value": "15"
                },
                {
                    "title": "30 minutes",
                    "value": "30"
                }
            ]
        },
        {
            "type": "ActionSet",
            "actions": [
                {
                    "type": "Action.OpenUrl",
                    "title": "Action.OpenUrl",
                    "url": "https://microsoft.com/"
                }
            ]
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "title": "Snooze",
            "data": {
                "x": "snooze"
            }
        },
        {
            "type": "Action.Submit",
            "title": "I'll be late",
            "data": {
                "x": "late"
            }
        }
    ]
}
```
